File: sp2mic-backend/sp2mic.WebAPI/Modulo2.Analise/Dtos/AtributoDto.cs
using sp2mic.WebAPI.Domain.Enumerations;

namespace sp2mic.WebAPI.Modulo2.Analise.Dtos;

public class AtributoDto
{
  public int? Id {get; set;}
  public string? NoAtributo {get; set;}
  public string? NoTipoDado {get; set;}
  public TipoDadoEnum? CoTipoDado {get; set;}
  public int? IdDtoClasse {get; set;}
}

File: sp2mic-frontend/apps/sp2mic/src/app/shared/models/atributo-dto.ts
import { DtoClasse } from './dto-classe';
import { TipoDadoEnum } from '../enums/tipo-dado-enum';

export class AtributoDto {
  id?: number;
  noAtributo?: string;
  coTipoDado?: TipoDadoEnum;
  noTipoDado?: string;
  idDtoClasse?: number;
  //noDtoClasse?= "";
  noMicrosservico?: string;
  dtoClasse?: DtoClasse;
}

export class AtributoDtoPaginado extends AtributoDto {
  page: number;
  linesPerPage: number;
  orderBy: string;
  direction: string;
}
